arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/Hub.java

```diff
@@ -1006,6 +999,10 @@ void processStateExec(MessageFrame frame) {
 
       new StackOnlySection(this);
     }
+
+    if (Exceptions.any(pch().exceptions()) || opCode() == REVERT) {
```

Comment:
we were setting the revert stamp BEFORE creating the section, and as we increase the hubStamp when creating the section, we gave h-1 for the revert stamp, not h.

Reply:
Nice